Dockerfile

```diff
@@ -1,46 +1,95 @@
-FROM        registry.access.redhat.com/ubi9/python-39:9.5-1739799514@sha256:9bbc4cfeac896544ab3eafa088c3d6995e82592362d374606f00d221f2986fe0 as prod
+##############
+# base image #
+##############
+FROM registry.access.redhat.com/ubi9/python-39:9.5-1739799514@sha256:9bbc4cfeac896544ab3eafa088c3d6995e82592362d374606f00d221f2986fe0 as base
 
+COPY        LICENSE /licenses/LICENSE
+
+#################
+# builder image #
+#################
+FROM base as builder
+
+# Get the uv binary from upstream
 COPY --from=ghcr.io/astral-sh/uv:0.6.11@sha256:fb91e82e8643382d5bce074ba0d167677d678faff4bd518dac670476d19b159c /uv /bin/uv
```

Comment:
please bump to latest uv
```dockerfile
COPY --from=ghcr.io/astral-sh/uv:0.7.13@sha256:6c1e19020ec221986a210027040044a5df8de762eb36d5240e382bc41d7a9043 /uv /bin/uv
```

Reply:
> the build arg broke `docker compose up`
> 
> ```
> failed to solve: failed to parse stage name "${UV_SRC_IMAGE}": invalid reference format: repository name (library/${UV_SRC_IMAGE}) must be lowercase
> ```

This seems like a bug in docker-compose. Build args are standard in `Dockerfile`
 
> renovate can detect image tag used in `COPY`, example pr [app-sre/python-project-template#99](https://github.com/app-sre/python-project-template/pull/99), so you can keep it as is like template https://github.com/app-sre/python-project-template/blob/78417dfcf8b95cce3b772df331d50159f8f4f742/%7B%7B%20cookiecutter.project_slug%20%7D%7D/Dockerfile#L14

🤔 I see. I suppose we'll use the bots to keep this updated.